### DISCUSSION TITLE
File format considerations

### DISCUSSION THREAD

**Discussion by mbasaglia** [lottie/lottie-spec#4] [Ideas]
Most file formats have a unique extension and mime type. Currently lottie files are indistinguishable from plain JSON. Should we recommend a unique file extension / register a new mime-type?

**Comment by fmalita** [comment]
`.lot` and `.lottie` come to mind, but the former seems taken (LaTeX List Of Tables) - so maybe the latter?

Should we worry about encodings other than plain JSON, say BSON or other/future binary encodings?  I guess if those ever come up we could use some extension variations (`.lottieb`, etc.).

**Reply by mbasaglia** [reply]
`.lottie` is used as file extension by dotLottie.

I think a binary format could be useful, I believe CBOR is more standardized than BSON. Or perhaps we could design something custom on top of RIFF to simplify parsing. But probably this is something we don't need to worry until after we finalize the JSON specs.
